docs/game_sdk/Applications.md

```diff
@@ -134,19 +134,19 @@ applicationManager.ValidateOrExit((result) =>
 ###### Current Version
 
 | version | status  |
-| ------- | ------- |
+|---------|---------|
 | 2       | current |
 
 Get the signed app ticket for the current user. The structure of the ticket is: `version.signature.base64encodedjson`, so you should split the string by the `.` character. Ensure that the `version` matches the current version. The `signature` is used to verify the ticket using the libsodium library of your choice, and the `base64encodedjson` is what you can transform after verification. It contains:
 
 - the application id tied to the ticket
-- the user's user id
+- the user's user-id
```

Comment:
This is pretty strange wording in general here. Not sure if it would be best to just cut out the redundant second `user`.